README.md

```diff
@@ -120,11 +120,11 @@ Everything you need is available prebuild in FreeBSD' ports.
 
 You can install cjdns by running
 
-    pacman -S cjdns
+    pacman -Sy cjdns
```

Comment:
What the fuck.
-Sy is dangerous! Never tell people to use it!
http://gist.io/5660494